src/prepare_pcap.c

```diff
@@ -545,6 +545,12 @@ int prepare_dtmf(const char* digits, pcap_pkts* pkts, uint16_t start_seq_no)
     unsigned long ts_offset = 0; /* packet timestamp */
     unsigned timestamp_start = 24000; /* RTP timestamp, should be random */
 
+    /* generate random ssrc */
+    if (dtmf_ssrcid == 0) {
+        srand(time(NULL));
```

Comment:
I don't think we want to call srand() here. It will reset the random. Rather call it at the start somewhere.

Reply:
Done by the original poster.